comp/core/remoteagentregistry/impl/remoteagentregistry.go

```diff
@@ -126,11 +210,14 @@ func (ra *remoteAgentRegistry) RegisterRemoteAgent(registration *remoteagentregi
 		// connecting when we try to query the remote agent for status or flare data.
 		details, err := newRemoteAgentDetails(registration)
 		if err != nil {
+			ra.telemetryStore.remoteAgentRegisteredError.Inc(registration.DisplayName, err.Error())
```

Comment:
I think we'd want to be using `agentID`, since display name will be stuff like "Agent Data Plane", etc... which is hard to use as a tag value for querying purposes.

Reply:
I don't have a string opinion about this, but looking at the RFC for the new Remote Agent Registry, it seems that the agentID might be remove from the registration payload https://datadoghq.atlassian.net/wiki/spaces/ARUN/pages/5262901901/RFC+Future-proofing+the+RemoteAgentRegistry#Removed-items

If the ID is removed, we can sanitize the display name